maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/HardshipOrchestrationServiceTest.java

```diff
@@ -93,7 +94,10 @@ private WorkflowRequest setupCreateStubs(CourtType courtType, CurrentStatus stat
         when(assessmentSummaryService.getSummary(any(HardshipReviewDTO.class), eq(courtType)))
                 .thenReturn(getAssessmentSummaryDTO());
 
-        doReturn(Boolean.TRUE).when(validationService).isUserActionValid(any());
+        when(hardshipMapper.getUserActionDTO(any(), any()))
+                .thenReturn(UserActionDTO.builder().username("mock-u").build());
+
+        doReturn(Boolean.TRUE).when(validationService).isUserActionValid(any(), any());
```

Comment:
It would be good if we could take the opportunity to do a bit of cleanup here. The `doReturn.when` can be used for partial mocking, which isn't what we're doing here, so I'd suggest changing these to the usual `when.thenReturn`. This also applied to `doThrow` and `when.thenThrow`.

Reply:
Happy to do a bit of refactoring here to clean up the existing code 👍

Reply:
Done